### PR TITLE
major release

### PR DESCRIPTION
**Pull request by rociojoo**

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,3 +1,90 @@
...
+## Packages removed
+
+4 packages have been removed from the list:
+
+  * [`Rhabit`](https://github.com/cran-task-views/Tracking/issues/20): does not 
+  pass checks anymore
+  * [`animalTrack`](https://github.com/rociojoo/CranTaskView-Track/issues/17): no
+    longer on CRAN (archived on 2023-02-02).
+  * [`argosTrack`](https://github.com/cran-task-views/Tracking/issues/60): does 
+  not pass checks anymore
```
Comment (basille) [inline NEWS.md:73]:
```suggestion
  not pass checks anymore
  * [`BayesianAnimalTracker`](https://github.com/cran-task-views/Tracking/issues/48): no
    longer on CRAN (archived on 2022-06-08)
  * [`BBMM`](https://github.com/cran-task-views/Tracking/issues/46): no
    longer on CRAN (archived on 2022-05-23)
```

File: NEWS.md [hunk 2]
```diff
@@ -1,3 +1,90 @@
...
+  * [`foieGras`](https://github.com/rociojoo/CranTaskView-Track/issues/57): no
+    longer on CRAN (archived on 2022-12-12).
```
Comment (basille) [inline NEWS.md:75]:
```suggestion
    longer on CRAN (archived on 2022-12-12)
  * [`mkde`](https://github.com/cran-task-views/Tracking/issues/47): no
    longer on CRAN (archived on 2022-04-25).
```

**Review by basille — CHANGES_REQUESTED** [review]
I've finished the review. I cleaned all GitHub issues, added a few archived packages, and added a comment about actual check results… This needs to be addressed before merging.

File: NEWS.md [hunk 3]
```diff
@@ -1,3 +1,90 @@
...
+  And Light Data In R
+  * [`smoove`](https://github.com/cran-task-views/Tracking/issues/41): 
+  Simulation and Estimation of Correlated Velocity Movement (CVM) Models
+  * [`vmsbase`](https://github.com/cran-task-views/Tracking/issues/43): 
+  GUI Tools to Process, Analyze and Plot Fisheries Data
+
+
+## Packages removed
+
+4 packages have been removed from the list:
```
[the remark this answers is not in the document]
Comment (rociojoo) [reply NEWS.md:69]: Thanks. Now they do.

**Pull request merged**